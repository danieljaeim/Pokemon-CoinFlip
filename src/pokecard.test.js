import React from "react";
import { shallow, mount } from "enzyme";
import toJson from "enzyme-to-json";
import Pokecard
  from "./pokecard";

it("renders without crashing", function() {
  shallow(<Pokecard />);
});

xit("matches snapshot", function() {
  let wrapper = mount(<Pokecard />);
  let serialized = toJson(wrapper);
  expect(serialized).toMatchSnapshot();
});