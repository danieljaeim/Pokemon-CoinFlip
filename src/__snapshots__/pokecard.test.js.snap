// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<Pokecard>
  <div
    style={
      Object {
        "backgroundColor": undefined,
        "borderRadius": "5px",
        "display": "inline-block",
        "margin": "30px",
        "padding": "25px",
      }
    }
  >
    <p>
      <h3 />
    </p>
    <img
      alt="this img"
      src="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/undefined.png"
    />
    <p>
      Type: 
    </p>
    <p>
      EXP: 
    </p>
  </div>
</Pokecard>
`;
